### PR TITLE
update description and news.md for v1.5

### PR DESCRIPTION
**Pull request by rvitancol**

### CODE REVIEW

**Comment by maddygriz** [comment]
can you do `devtools::document()`? We need to update the NAMESPACE with new dependencies.

**Comment by maddygriz** [comment]
In that case, can you manually add the rfast functions used?

**Comment by rvitancol** [comment]
pls check Maddy, and merge when you think it is acceptable.

File: NAMESPACE [hunk 1]
```diff
@@ -1,19 +1,38 @@
 useDynLib(GeoDiff, .registration=TRUE)
 importFrom(Rcpp, evalCpp)
+importFrom(Rcpp,sourceCpp)
 
 
 import(Biobase)
```
Comment (maddygriz) [inline NAMESPACE:6]: we probably need either `import(Biobase)` or all the `importFrom(Biobase,...)` lines, not sure which one is better

**Comment by maddygriz** [comment]
I added just a couple documentation things but it looks good

**Review by maddygriz — APPROVED** [review]

File: NAMESPACE (hunk 1 above)
Comment (rvitancol) [reply NAMESPACE:6]: > we probably need either `import(Biobase)` or all the `importFrom(Biobase,...)` lines, not sure which one is better

I removed the others.

**Pull request merged**